src/main/webapp/WEB-INF/pages/productList.jsp

```diff
@@ -8,22 +8,40 @@
   <p>
     Welcome to Expert-Soft training!
   </p>
+  <form>
+    <input name="query">
```

Comment:
Please make sure the value is displayed - please try: http://localhost:8080/phoneshop-servlet-api/products?sort=description&order=asc&query=Samsung